aether-couchdb-sync-module/conf/check_vars.sh

```diff
@@ -38,3 +38,6 @@ check_variable $AETHER_KERNEL_TOKEN "Aether KERNEL token (AETHER_KERNEL_TOKEN)"
 
 # Google ID
 check_variable $GOOGLE_CLIENT_ID    "Google Client ID (GOOGLE_CLIENT_ID)"
```

Comment:
Can we quote these (i.e. `"GOOGLE_CLIENT_ID"`) as well? In my bash version, this script exits with `0` even though none of those env vars are set. Adding quotes fixes that.